doc/specifications/translations/removal.md

```diff
@@ -14,7 +14,7 @@ the existing Versions of a Content Object and is specified as:
  *       a warning before performing it.
  *
  * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException if the specified translation
- *         is the only one Content Object has.
+ *         is the only one Content Object has or it is the main translation.
```

Comment:
On the initial version I might be wrong, but afaik the current proposed logic means it won't allow you to delete any language, as typically there will be one version for each translation added where it is the initial one.